src/udev/udevadm-info.c

```diff
@@ -250,7 +250,6 @@ static void help(void) {
         printf("%s info [OPTIONS] [DEVPATH|FILE]\n\n"
                "Query sysfs or the udev database.\n\n"
                "  -h --help                   Print this message\n"
-               "  -V --version                Print version of the program\n"
```

Comment:
This kind of change always requires a commit message!
So this patch removes `udevadm info --version`, but `udevadm --version info` still works.
I'm not sure if this change is warranted: can you provide some justification why that's wanted?

Reply:
Hmm. Not really wanted. Just for the consistency to other sub-commands. No other sub-command has `--version` option.

Reply:
Or should be `--version` added to the other sub-commands?

Reply:
Well... consistency is good, but it shouldn't override everything else. Personally, I think that it's a crappy parser which treats `prog --option verb` different from `prog verb --option`. We *do* have a crappy parser in udevadm, but this change makes things worse. Another consideration is that this breaks backwards compatibility, somebody out there might be using `udevadm info --version` in a script to check udev version.

I'm against removing the option.